### PR TITLE
[BEAM-10306] Add latency measurements to Python Flink load test 

### PR DESCRIPTION
**Pull request by mxm**
There are currently no latency metrics in the load tests which makes it impossible to monitor latency regressions.

This change adds latency to the existing benchmarks.

Dashboard with latency results: https://apache-beam-testing.appspot.com/explore?dashboard=5751884853805056

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Dataflow | Flink | Samza | Spark
--- | --- | --- | --- | --- | ---
Go | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Java11/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming/lastCompletedBuild/)
Python | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow_V2/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Python2_PVR_Flink_Cron/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python35_VR_Flink/lastCompletedBuild/) | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://ci-beam.apache.org/job/beam_PostCommit_XVR_Spark/lastCompletedBuild/)

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/)<br>[Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/)
Portable | --- | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by mxm** [comment]
Run Seed Job

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run Python2_PVR_Flink PreCommit

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run Python PreCommit

**Comment by mxm** [comment]
Run Python2_PVR_Flink PreCommit

**Comment by mxm** [comment]
Run Seed Job

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run Python PreCommit

**Comment by mxm** [comment]
Run Python PreCommit

**Comment by mxm** [comment]
Run Seed Job

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run Seed Job

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run PythonFormatter PreCommit

**Comment by mxm** [comment]
Run Seed Job

**Comment by mxm** [comment]
Run Python Load Tests ParDo Flink Streaming

**Comment by mxm** [comment]
Run Python2_PVR_Flink PreCommit

**Comment by mxm** [comment]
Run PythonFormatter PreCommit

**Comment by mxm** [comment]
PythonFormatter / Python PreCommit are broken at the moment, but it's unrelated to the changes here.

File: sdks/python/apache_beam/testing/load_tests/pardo_test.py [hunk 1]
```diff
@@ -147,10 +151,18 @@ def process(self, element, state=state_param):
               CounterOperation(
                   self.number_of_counters, self.number_of_operations)))
 
+    if self.window:
+      pc = (
+          pc
+          | beam.WindowInto(beam.window.FixedWindows(1))
+          | beam.GroupByKey())
```
Comment (kamilwu) [inline sdks/python/apache_beam/testing/load_tests/pardo_test.py:158]: Just out of curiosity, could you explain, why do we need GroupByKey in this place?

Comment (mxm) [reply sdks/python/apache_beam/testing/load_tests/pardo_test.py:158]: The goal here is to test GroupByKey which holds back windowed values based on the watermark. This creates additional latency. If we do not use GroupByKey, then there is a simple pass-through of all elements without any buffer.

**Comment by mxm** [comment]
Run PythonFormatter PreCommit

**Comment by mxm** [comment]
Run Python PreCommit

File: sdks/python/apache_beam/testing/load_tests/pardo_test.py (hunk 1 above)
Comment (kamilwu) [reply sdks/python/apache_beam/testing/load_tests/pardo_test.py:158]: I see. In this case, maybe it'd would worth to measure latency in GroupByKey Load Test: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/testing/load_tests/group_by_key_test.py? GroupByKey step is already there.

Comment (mxm) [reply sdks/python/apache_beam/testing/load_tests/pardo_test.py:159]: Good point, I think we could move it there.

Comment (mxm) [reply sdks/python/apache_beam/testing/load_tests/pardo_test.py]: I've removed the GroupByKey here and added a latency measurement to the existing GroupByKey test.

**Comment by mxm** [comment]
I've removed the GroupByKey from the ParDo test and instead added latency measurements to the dedicated GroupByKey load test. So both ParDo and GroupByKey measure latency now.

**Comment by mxm** [comment]
Run Seed Job

**Pull request merged**